styles/signals.mapcss

```diff
@@ -69,10 +69,18 @@ way["railway:lzb"=yes].tracks
 	z-index: 3;
 	color: red;
 }
+way["railway:atb"=yes].tracks
```

Comment:
This is not doing what you expect it to do. You need a comma at the end of the line.

Reply:
Added commas to the line-ends... Is that correct?

Reply:
Yes, looks good now.